example/flake/flake.nix

```diff
@@ -7,7 +7,7 @@
   inputs = {
     flake-utils.url = "github:numtide/flake-utils";
     nixpkgs.url = "nixpkgs/a5d03577f0161c8a6e713b928ca44d9b3feb2c37";
-    jupyterWith = { url = "path:../../."; };
+    jupyterWith.url = github:tweag/jupyterWith;
```

Comment:
had to change it else when starting the flake from the example/flake folder it gives `relative path '../../.' points outside of its parent's store path '/nix/store/j6hpzfy2zi7pm8pxv1l61h1j4v24mpfc-source'`